src/docker-osx-dev

```diff
@@ -226,9 +225,9 @@ function do_rsync {
 #
 function sync {
   local readonly paths_to_sync=("$@")
-
-  for path in "${paths_to_sync[@]}"; do
-    do_rsync "$path"
+  for pathRsync in ${paths_to_sync[@]}
```

Comment:
For the testing purpose, I was just thinking if the var was not global.

This does correct the issues #38 : 
"${paths_to_sync[@]}" => with this the" for" would juste have the last.
${paths_to_sync[@]} => it will initilly sync all the folders présent in docker-comose.yml